Utils/main.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+
+from helper_functions import find_median, open_file, plot_histogram
+
+
+# define the main function for the program to run from here
+def main(
```

Comment:
Consider giving this function a more opinionated name which is associated with the action it provides. As a general practice, I recommend using verbs in some way to help with maintainer understanding about the movement or change functions grant to the work.